rmd/Story 1.json

```diff
@@ -66,17 +66,17 @@
   },
   "100010020040008": {
     "OriginalText": "私はアークス特殊惑星探査隊オルター所属艦、\nデルタ・ヴァリアントのメディカルスタッフ、\nリーティアと申します。",
-    "Text": "We are on the ARKS Special Planetary Exploration\nGroup 'Alter' ship, \"Delta Valiant\". I'm Rietheia,\npart of the medical staff.",
+    "Text": "This ship is called the \"Delta Valiant\", part of the ARKS Special Planetary Exploration Team, \"Alter\". My name is Rietheia, part of the medical staff.",
```

Comment:
This is too long? the script check if it longer then 110 chars

Reply:
I've never personally experimented with the line lengths of the story dialogues but I imagine they're smaller than that, knowing how limited everything else is. I wonder if it's even a good idea to enforce line limits via testing without even knowing what they all are yet.